src/lib/env.ts

```diff
@@ -1,6 +1,7 @@
 export const ENV = {
   OS_RAISE_CONTENT_URL_PREFIX: import.meta.env.MODE === 'production' ? 'https://k12.openstax.org/contents/raise' : 'http://localhost:8800/contents',
   OS_RAISE_EVENTSAPI_URL_MAP: {},
+  OS_RAISE_ENABLE_MOODLE_PERSISTOR_MAP: {},
```

Comment:
Can we slightly modify this to be a list? I think it will make things simpler:

```javascript:
OS_RAISE_ENABLE_MOODLE_PERSISTOR_HOSTS: [] as string[],
```